identifiers/country-in/aliases.csv

```diff
@@ -1,546 +1,549 @@
-id,sameAs,sameAsNote
```

Comment:
Any idea what's going on with this file? It looks like just the last 3 lines were added. Ideally the diff would just be that so that the version history is easier to browse.

Reply:
No idea, I copied the raw text from the original file vs the modified one into a diff tool, and only the last 3 lines are highlighted as the diff.
![Screenshot of diff](https://user-images.githubusercontent.com/60165975/121986181-a9bfba80-cdc8-11eb-9b0f-393727f1fa0d.png)

Reply:
Strange, my only guess is that maybe the line terminators changed, one might have had `CR LF` and the other just `LF`. Either way I don't think it's necessarily blocking.